facia/app/controllers/FaciaController.scala

```diff
@@ -25,6 +25,14 @@ trait FaciaController extends Controller with Logging with ExecutionContexts wit
 
   val frontJsonFapi: FrontJsonFapi
 
+  private def getEditionFromString(edition: String) = {
+    val editionToFilterBy = edition match {
+      case "international" => "int"
+      case _ => edition
+    }
+    Edition.all.find(_.id.toLowerCase() == editionToFilterBy).getOrElse(Edition.all.head)
+  }
+
```

Comment:
This may not be needed, as we get the edition from the request anyway. We should test it without and see whether it works. If it does, we can remove this completely.